source/dbo/Stored Procedures/Power BI Reporting/uspGenerateReportingLegacySitesOfDisease.sql

```diff
@@ -64,6 +64,17 @@ BEGIN TRY
 		FROM [dbo].[Record_LegacyExtract] le
 			INNER JOIN ctePivotResults pr ON pr.NotificationId = le.NotificationId
 
+	--Add in the OtherExtraPulmonarySite text
+	UPDATE le
+		SET OtherExtraPulmonarySite = Descriptions.SiteDescription
+	FROM [dbo].[Record_LegacyExtract] le
+		INNER JOIN (
+			SELECT NotificationId,
+					SiteDescription
+				FROM [$(NTBS)].[dbo].[NotificationSite]
+				WHERE SiteId = 17
+		) AS Descriptions ON Descriptions.NotificationId = le.NotificationId
```

Comment:
This is probably just a personal preference, but I'd find this clearer without the nested select, i.e.
```
UPDATE le
SET OtherExtraPulmonarySite = ns.SiteDescription
FROM [dbo].[Record_LegacyExtract] le
INNER JOIN [$(NTBS)].[dbo].[NotificationSite] ns ON ns.NotificationId = le.NotificationId
WHERE ns.SiteId = 17
```

Reply:
That said, I don't know which would be more performant

Reply:
Ah that definitely would be tidier and shouldn't impact performance at all. I have just merged this though, so I'll make a quick follow on PR

Reply:
Follow-on PR at #69 @JoshCadney